etl/steps/data/garden/un/2024-07-08/maternal_mortality.meta.yml

```diff
@@ -0,0 +1,148 @@
+        and a 50% chance that the true value lies below this point.
+  description_hiv: |-
+    For countries with generalized HIV epidemics and high HIV prevalence, HIV/AIDS is a leading cause of death during pregnancy and after delivery.
+    Furthermore, evidence suggests that women with HIV infection have an eight times higher risk of pregnancy-related death compared with non-HIV infected women.
+
+    When estimating maternal mortality in these countries, it is, thus, important to differentiate between incidental HIV deaths (non-maternal deaths) and HIV-related indirect maternal deaths (maternal deaths caused by the aggravating effects of pregnancy on HIV) among HIV-positive pregnant and postpartum women who have died.
+  description_methodology: |-
+    Data on maternal mortality and other relevant variables are obtained through databases maintained by WHO, UNPD, UNICEF, and the World Bank Group. Data available from countries varies in terms of the source and methods.
+
+    Given the variability of the sources of data, different methods are used for each data source in order to arrive at country estimates that are comparable and permit regional and global aggregation.
+
+    The current methodology employed by the Maternal Mortality Estimation Inter-Agency Group (MMEIG) in this round
+    followed an improved approach that built directly upon methods used to produce the previous
+    rounds of estimates published by the MMEIG since 2008.
+
+    Estimates for this round were generated using a Bayesian approach, referred to as the Bayesian maternal mortality estimation model, or BMat model.
+    This enhanced methodology uses the same core estimation method as in those previous rounds, but adds refinements to optimize the use of country-specific data sources and excludes late maternal deaths.
+    It therefore provides more accurate estimates, and a more realistic assessment of certainty about those estimates.
+
+    The new model still incorporates the same covariates which are;
+    - the Gross Domestic Product per capita based on purchasing power parity conversion (GDP),
+    - the general fertility rate (GFR)
+    - proportion of births attended by a skilled health worker (SAB).
+
+    The MMEIG has developed a method to adjust existing data in order to take into account these data quality issues and ensure the comparability of different data sources.
+    This method involves assessment of data for underreporting and, where necessary, adjustment for incompleteness and misclassification of deaths as well as development of estimates through statistical modelling for countries with no reliable national level data.
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  maternal_mortality:
+    variables:
+      births:
+        title: Estimated live births
+        description_short: The number of live births in a given year.
```

Comment:
Perhaps worth doubling down on the 'estimated' with "The estimated number of live births in a given year.'

Reply:
true! I'll include it!